spec_cn.md

```diff
@@ -548,11 +547,11 @@ curl -X GET http://localhost:8081/subjects/test-value/versions/1/schema
 	"app": "rocketmq",
 	"description": "rocketmq user information",
 	"compatibility": "NONE",
+	"schemaType": "AVRO",
```

Comment:
Confused when we pull schemaType out of schema.

Reply:
I would like to suggest to define the schemaType such as JSON, AVRO in Subject level. The reason is to restrict all schema versions under the same Subject should accommodate to the same schemaType such as JSON or AVRO.  If user needs to create different schemaTypes , user can register multiple Subjects against each schemaType such as JSON or AVRO.